pages/common/fls.md

```diff
@@ -0,0 +1,16 @@
+# fls
+
+> List files and directories in an image file or device.
+> More information: <https://wiki.sleuthkit.org/index.php?title=Fls>.
+
+- Build a recursive fls list over a device:
+
+`fls -r -m {{original_path}} {{/dev/examplepartition}}`
+
+- Analyse a single partition, providing the timezone of the original system and the sector offset at which the file system starts in the image:
+
+`fls -r -m {{original_path}} -z {{timezone}} -o {{sector}} {{imagefile}}`
```

Comment:
Since we have more space, we can gradually introduce the options `-z` and `-o` one by one.

Also, let's change imagefile to path/to/image.